app/containers/Navbar/HeroNav.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React from 'react'
+import classNames from 'classnames'
+import styles from './styles.css'
+import LogoInvert from 'static/img/logo/studs19_invert.svg'
+import Logo from 'static/img/logo/studs19.svg'
+import MenuIcon from './icon_menu.svg'
+import BlackMenuIcon from './icon_menu_black.svg'
+import CloseIcon from './icon_close.svg'
+import BlackCloseIcon from './icon_close_black.svg'
+import Isvg from 'react-inlinesvg'
+
+import { withRouter, Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+import NavLinks from './NavLinks'
+
+type Props = {
```

Comment:
Typically we want to do:

```js
type Props = {|
//...
|}
```

Why? Because with `|` we tell Flow we want an exact match of the object (thus we only allow props that have been declared). We can also prefix the props with `+` to declare them read-only.